.github/workflows/ports_syscfg_check.yml

```diff
@@ -58,15 +58,18 @@ jobs:
         run: |
              cd build
              ./repos/apache-mynewt-nimble/porting/update_generated_files.sh
-             cd repos/apache-mynewt-nimble
+             cd ..
```

Comment:
there's no point in changing working directory at the end of script

Reply:
Agreed, but that was what @sjanc suggested for this script to be more future-proof. Should I remove these 'cd's at the end of each step?